pkg/cmd/openshift/openshift.go

```diff
@@ -99,13 +100,11 @@ func NewCommandOpenShift(name string) *cobra.Command {
 		Run:   kcmdutil.DefaultSubCommandRun(out),
 	}
 
-	f := clientcmd.New(pflag.NewFlagSet("", pflag.ContinueOnError))
-
 	startAllInOne, _ := start.NewCommandStartAllInOne(name, out, errout)
 	root.AddCommand(startAllInOne)
 	root.AddCommand(newExperimentalCommand("ex", name+" ex"))
 	root.AddCommand(newCompletionCommand("completion", name+" completion"))
-	root.AddCommand(cmd.NewCmdVersion(name, f, out, cmd.VersionOptions{PrintEtcdVersion: true, IsServer: true}))
+	root.AddCommand(cmdversion.NewCmdVersion(name, version.GetWithEtcdVersion(), os.Stdout))
```

Comment:
show me the output of `openshift version` before and after please.

Reply:
**Before**
```
$ openshift version
openshift v3.8.0-alpha.0+da1cca8-325-dirty
kubernetes v1.8.1+0d5291c
etcd 3.2.8
```

**After**
```
$ openshift version
openshift v3.8.0-alpha.0+9d4c131-327-dirty
etcd 3.2.8
```

Per https://github.com/openshift/origin/pull/17357#discussion_r153891196, will make a new version command for `openshift` to address this discrepancy